fastlane/Fastfile

```diff
@@ -154,13 +154,18 @@ platform :android do
     # This dependency could be removed with a little more time to tidy up this script to do the branching/merging manually.
 
     desc "Create new release"
-    lane :release do
+    lane :release do |option|
```

Comment:
```suggestion
    lane :release do |options|
```

Reply:
This change is required, otherwise running `bundle exec fastlane release` will break complaining about it (as it tries to access non-existent option**s**

Reply:
You are right. I mistyped this one. Good catch!